test/unit/components/messages/message.test.js

```diff
@@ -0,0 +1,61 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var expect = chai.expect;
+
+var Message = require('../../../../app/components/messages/message');
+
+describe('Message', function () {
+  describe('getInitialState', function() {
+    it('should return an object with editing set to false', function() {
+      console.warn = sinon.spy();
+      // TODO: consider requiring Proptype: theNote (maybe define as a shape as well?)
+      //   -> Properties of theNote are accessed in componentDidMount()
```

Comment:
Feel free to address this TODO in this PR. Generally we do prefer `isRequired` on our props, but clearly there's some inconsistency across the codebase... Here I think we should be OK (i.e., without any other code changes) adding `isRequired` on `theNote` and `onSaveEdit`; possibly also on `imageSize`.